roles/rundeck/defaults/main.yaml

```diff
@@ -2,14 +2,20 @@
 rundeck_enabled: yes
 rundeck_prune_dangling: yes
 
-rundeck_container_env_file: /usr/local/percona/rundeck/rundeck_env.cfg
-rundeck_container_realm_file: /usr/local/percona/rundeck/rundeck_realm.properties
+rundeck_container_env_file: /usr/local/percona/rundeck/env.cfg
+rundeck_container_realm_file: /usr/local/percona/rundeck/realm.properties
 rundeck_container_extra_args: '--ulimit=host -p 127.0.0.1:4440:4440 --volume /dev/null:/home/rundeck/etc/rundeckpro-license.key --volume {{ rundeck_container_realm_file }}:/home/rundeck/server/config/realm.properties --volume {{ rundeck_container_volume }}:/home/rundeck/server/data'
 rundeck_container_image_repo: '{{ "docker.io/rundeck" }}'
 rundeck_container_port: 4440
 rundeck_container_volume: rundeck-data
 
 rundeck_deploy_using: '{{ container_engine | mandatory }}'
 
-rundeck_version: 4.7.0
+# CHANGE DEFAULT CREDENTIALS IN OVERRIDES OR SECRET FILE BEFORE DEPLOYMENT
```

Comment:
Please remove this comment, it is inaccurate. In addition, a change to the secrets in the realm properties should trigger a change to the Rundeck container and so it doesn't matter when they are set. 

Also, please do not use UPPERCASE as it is harder to read